Cargo.lock

```diff
@@ -2,15 +2,6 @@
 # It is not intended for manual editing.
 version = 4
 
-[[package]]
```

Comment:
Question: Why was the `Cargo.lock` updated as part of a bug fix? Were updated dependencies required?

For context, our dependent project generally follows a cooldown period for upgraded dependencies to minimise the risk of undetected Shai-hulud attacks, making exceptions where necessary. Our builds are broken due to the `svm-build` bug that this PR proposes to fix, so that would warrant an exception for `svm-rs`. However, ideally we'd avoid additional dependency upgrades.

Reply:
looks like cargo update @mablr we can exclude from this pr, but since this is a lib we could remove the lockfile from the repo entirely, separately